bridge/client/webrtc.js

```diff
@@ -795,8 +806,12 @@
                 if (!mdesc.ice) {
                     console.warn("setRemoteDescription: m-line " + i +
                         " is missing ICE credentials");
-                    mdesc.ice = {};
+                    mdesc.ice = {
+                        "iceOptions": {}
+                    };
                 }
+                if (mdesc.ice.iceOptions.trickle)
+                    canTrickle = true;
```

Comment:
It feels a bit strange to have this option for each m-line. Isn't trickling a feature that's supported by the endpoint? I guess it works if all m-lines are marked consistently. I'm asking because of the above code that sets the RTCPeerConnection attribute if a single m-line has the option.